docs/sdks/client-sdks/react-native/initialization.mdx

```diff
@@ -67,6 +67,15 @@ The base URL for for the API. Can be overriden to proxy configuration requests.
 </ApiOptionRef>
 
 
+<ApiOptionRef 
+  name="pollAfterSuccessfulInitialization"
```

Comment:
Proof here that `pollAfterSuccessfulInitialization` is used by the react native SDK

 https://github.com/Eppo-exp/react-native-sdk/blob/96dcd2aebbf687d2804aec1bfef052c585f30f08/src/index.tsx#L103-L104